web/pgadmin/static/js/components/PgTree/FileTreeItem/DoubleClickHandler.jsx

```diff
@@ -15,7 +15,12 @@ import CustomPropTypes from '../../../../js/custom_prop_types';
 export default function DoubleClickHandler({onSingleClick, onDoubleClick, children}){
   const onClick = useSingleAndDoubleClick(onSingleClick, onDoubleClick) ;
   return(
-    <div onClick={(e)=>onClick(e)}>
+    <div onClick={(e)=>onClick(e)} onKeyDown={(e)=>{
```

Comment:
how about we create a wrapper like -
```
function getEnterHandler(callback) {
    return (e)=>{
        if(e.code === 'Enter'){
            e.preventDefault();
            callback(e);
         }
    }
}
```
and then `onKeyDown = { getEnterHandler(onClick) }`

Reply:
Done.